supplementary/alica_ros1/alica_tests/libalica-tests/package.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>libalica-tests</name>
+  <version>0.9.4</version>
+  <description>The alica test plan lib</description>
+
+  <maintainer email="stephan.opfer@rapyuta-robotics.com">Stephan Opfer</maintainer>
```

Comment:
Who should be added as a maintainer for these packages?

Reply:
Maybe just put maintainer/author as Rapyuta Robotics with no email field, if that is possible?